Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1006,6 +1006,9 @@ void RenderElement::styleWillChange(StyleDifference diff, const RenderStyle& new
         if (drawsRootBackground && newStyle.hasEntirelyFixedBackground() && view().compositor().supportsFixedRootBackgroundCompositing())
             newStyleSlowScroll = false;
     }
+    // FIXME: Scrolling tree support for anchor positioning.
+    if (newStyle.position() == PositionType::Fixed && Style::AnchorPositionEvaluator::isAnchorPositioned(newStyle))
```

Comment:
What about sticky positioning?

Reply:
Sticky can't be anchor positioned, only fixed and absolute.